homework.js

```diff
@@ -2,25 +2,53 @@
 // declare a variable called fruit
 // assign your variable the value 6
 // console log your variable
-
+let variable = fruit
+fruit = 6
+console.log(fruit)
 
 // write a function called product that accepts two numbers and 
 // logs the product in the console and then
 // returns the answer when you multiply them both together
 // call that function on the numbers 4 & 689 to test it
+function product(num, num2)
+console.log(product)
+return product(num*num2)
```

Comment:
You probably want to declare a variable 
`let answer = num*num2`
then you can console log and return the answer.
"Calling a function" means writing it's name with specific arguments.
`product(4, 689)